x-pack/legacy/plugins/ml/public/data_visualizer/common/field_vis_config.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ML_JOB_FIELD_TYPES } from '../../../common/constants/field_types';
+
+// The internal representation of the confuguration used to build the visuals
```

Comment:
Small type `configuration`

Reply:
Done in https://github.com/elastic/kibana/pull/42685/commits/31da8259e659eb908cba4922d4ab44bd0dd3898c